zio-http/shared/src/main/scala/zio/http/Response.scala

```diff
@@ -138,7 +138,7 @@ object Response {
 
     val message2 = OutputEncoder.encodeHtml(if (message == null) status.text else message)
 
-    Response(status = status, headers = Headers(Header.Warning(199, "ZIO HTTP", message2)))
+    Response(status = status, body = Body.fromString(message2))
```

Comment:
This does not offer an opt-in to this behaviour

Reply:
Fixed. Added a boolean flag to the method; let me know if there's an alternative way.